homeassistant/components/sensor/icons.json

```diff
@@ -9,6 +9,9 @@
     "aqi": {
       "default": "mdi:air-filter"
     },
+    "area": {
+      "default": "mdi:square"
```

Comment:
@epenet happy with square?

Reply:
Oh I like `texture-box`

Reply:
Ooh YES! that's perfect!

Reply:
At least under the vacuum and mowing integrations.

Reply:
Let's go for the `texture-box` then, it fits well now and it's very simple to change in the future if needed.